Remotion/Web/Development.WebTesting/HostingStrategies/Configuration/ITestSiteConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System.Collections.Generic;
+
+namespace Remotion.Web.Development.WebTesting.HostingStrategies.Configuration
+{
+  public interface ITestSiteConfiguration

```

Comment:
Naming issue:
Test*SITE* would imply the port-number as well. Maybe we need a better name for "Test*SITE*"  since *Site* is already a wellknown concept in web hosting that means more than a bunch of paths.

Reply:
What about `ITestSiteProjectConfiguration`? The contained path is the path to the project's directory, the needed resources are paths to required folders, not contained in the project.

Reply:
Hmm...no, can't really establish a connection to this name, it's a gut feeling right now, sorry! Am not sure we can find a good name doing the name finding online with ping pong.